### DISCUSSION TITLE
ARCHIVING: email storage

### DISCUSSION THREAD

**Discussion by dennyhalim** [modoboa/modoboa#1089] [Q&A]
can you please add email archiving feature?
something similar to this: https://github.com/andryyy/mailcow/wiki/Archiving-mail

thank you.

**Comment by tonioo** [comment]
@dennyhalim That's more a postfix feature than a modoboa one. You can achieve quick archiving using the `always_bcc` option.

**Comment by jficz** [comment]
Under certain circumstances such archiving can be achieved using Dovecot alternate storage and a simple cron script:

https://wiki2.dovecot.org/MailboxFormat/dbox#Alternate_storage

**Comment by km1** [comment]
I think the solution could be to make a process/service of copying mails to additional storage. Once the user is closing to his storage limit - or it could be a specific time e.g. older mail than 1 year - his mails are removed from primary mailbox. The "removed" mails are available only from "archive GUI" - could be something similar to Roundcube. 
I am not sure if there is such solution available.

**Comment by km1** [comment]
I found something like that: https://www.mailpiler.org Has anyone used it?
